ci/travis/cppcheck.sh

```diff
@@ -2,4 +2,6 @@
 
 set -e
 
-cppcheck --quiet --force --error-exitcode=1 $CPPCHECK_OPTIONS .
+[[ -f .cppcheckignore ]] && CPPCHECK_IGNORE="--suppressions-list=.cppcheckignore"
+	
+cppcheck --quiet --force --error-exitcode=1 $CPPCHECK_IGNORE .
```

Comment:
why not leave `CPPCHECK_OPTIONS` ? 
wasn't this used by some other repo ?

Reply:
We can leave the `CPPCHECK_OPTIONS` variable.
There is already a branch prepared for PR on the repo where this variable is used so that it fits the new changes.

Reply:
you could also do:
`[ ! -f .cppcheckignore ] || CPPCHECK_OPTIONS="--suppressions-list=.cppcheckignore $CPPCHECK_OPTIONS"`

this has 2 parts:
1. it re-uses `CPPCHECK_OPTIONS`
2. this evaluation always returns success; otherwise, using `&&` would fail the build in other repos